source/components/GlobalFooter/README.md

```diff
@@ -3,382 +3,372 @@ Defaults:
-           					"image-data": {
-           						"type": "wds-svg",
-           						"name": "wds-icons-instagram"
-           					},
-           					"title": {
-           						"type": "translatable-text",
-           						"key": "global-footer-follow-us-link-instagram"
-           					},
-           					"href": "https://www.instagram.com/getfandom/",
-           					"tracking_label": "follow-us.instagram"
-           				},
-           				{
-           					"type": "link-image",
-           					"image": "wds-icons-linkedin",
-           					"image-data": {
-           						"type": "wds-svg",
-           						"name": "wds-icons-linkedin"
-           					},
-           					"title": {
-           						"type": "translatable-text",
-           						"key": "global-footer-follow-us-link-linkedin"
-           					},
-           					"href": "https://www.linkedin.com/company/157252",
-           					"tracking_label": "follow-us.linkedin"
-           				}
-           			]
-           		},
-           		"community": {
-           			"header": {
-           				"type": "line-text",
-           				"title": {
-           					"type": "translatable-text",
-           					"key": "Community"
-           				}
-           			},
-           			"links": [
-           				{
-           					"type": "link-text",
-           					"title": {
-           						"type": "translatable-text",
-           						"key": "Community Central"
-           					},
-           					"href": "http:\/\/community.wikia.com\/wiki\/Community_Central"
-           				},
-           				{
-           					"type": "link-text",
-           					"title": {
-           						"type": "translatable-text",
-           						"key": "Support"
-           					},
-           					"href": "http:\/\/community.wikia.com\/wiki\/Special:Contact"
-           				},
-           				{
-           					"type": "link-text",
-           					"title": {
-           						"type": "translatable-text",
-           						"key": "Fan Contributor Program"
-           					},
-           					"href": "http:\/\/fandom.wikia.com\/fan-contributor"
-           				},
-           				{
-           					"type": "link-text",
-           					"title": {
-           						"type": "translatable-text",
-           						"key": "WAM Score"
-           					},
-           					"href": "http:\/\/www.wikia.com\/WAM"
-           				},
-           				{
-           					"type": "link-text",
-           					"title": {
-           						"type": "translatable-text",
-           						"key": "Help"
-           					},
-           					"href": "http:\/\/community.wikia.com\/wiki\/Help:Contents"
-           				}
-           			]
-           		},
-           		"is-wikia-org": false
-           	}}
+       "header": {
+            "type": "link-image",
+            "image-data": {
+                "name": "wds-company-logo-fandom-white"
+            },
+            "href": "http:\/\/fandom.wikia.com",
```

Comment:
I believe these `wikia` links are outdated. Could you please update in the README. Thanks.